models/biocro/tests/testthat/test.model2netcdf.BIOCRO.R

```diff
@@ -1,39 +1,37 @@
 context("check output from model2netcdf.BIOCRO")
 
-# library(data.table)
-# library(PEcAn.utils)
-# library(PEcAn.settings)
+library(data.table)
+library(PEcAn.utils)
+library(PEcAn.settings)
```

Comment:
is this necessary now that read.settings is called using `package::function()` notation?